pages/06_embedding_non_view_content.md

```diff
@@ -2,22 +2,64 @@
+
+1. Create a `<tableau-viz>` component (or `TableauViz` object), set the `suppress-default-edit-behavior` attribute to turn off the default actions that occur when the user clicks the **Edit** button, or closes, or publishes the view.
+
+1. In the `<tableau-viz>` component, set the `src` URL and then add the `onEditButtonClicked` event listener to call your custom handler (`handleEditButtonClicked()`) when the user wants to switch to edit mode.
+
+   ```html
+
+    <tableau-viz
+     id="viewingViz"
+     src="https://myserver/t/site/workbook/sheet"
+     width="800" height="600"
+     suppress-default-edit-behavior
+     onEditButtonClicked="handleEditButtonClicked">
+    </tableau-viz>
+
+   ```
+
+1. Create a `<tableau-authoring-viz>` component or `TableauAuthoringViz` object, but don't specify the `src` URL to the view, so that the view doesn't appear until after the **Edit** button is clicked. You can also set the HTML style properties to hide the component or `<div>` that will contain the authoring view.
+
+1. In your custom `handleEditButtonClicked()` method, you should assign the `src` URL to the `TableauAuthoringViz` object so that when it is rendered it shows the view. You should also set the style properties to show the authoring component and to hide the `TableauViz` object.
+
+1. In the `<tableau-authoring-viz>` component, add the `onWorkbookPublishedAs` event attribute or property to be able to get the new URL for the saved workbook. In your custom handler for the published-as event, assign the new URL to the `TableauViz` object. 
+
+1. In the `<tableau-authoring-viz>` component, add the `onWorkbookReadyToClose` event attribute to know when to switch back to view mode. In your custom event handler for the close event, hide the authoring component from view and show the viewing component.
+
+For an overview of the embedded web authoring feature, see [How to Enable Self-Service Analytics in Your Application with Embedded Web Authoring](https://www.tableau.com/about/blog/2022/8/how-enable-self-service-analytics-your-application-embedded-web-authoring). For a hands-on tutorial, see [Embedded API - Web Authoring Tutorial](https://www.tableau.com/developer/learning/embedding-api-web-authoring-tutorial). And for more information, see [Embedded Web Authoring](https://help.tableau.com/current/api/embedding_api/en-us/docs/embedding_api_web_authoring.html) in the Embedding API v3 Help documentation.
+
+
+<!-- Not sure if we need to keep any of the following. Does it still apply? -->
+// TO DO: Does the following still apply? 
```

Comment:
one and two are still accurate. We should remove 3 and replace 4 by Connected Apps and be sure to pass the right scope

Reply:
Thanks! 

* If you are using Connected Apps for embedding views, be sure to set the scope (`scp`) claim in the Jason Web Token (JWT) to include web authoring (`tableau:views:embed_authoring`). For information about Connected Apps, see [Authentication and Single Sign-On (SSO)]({{site.baseurl}}/pages/02_auth_and_sso) and [Configure Tableau Connected Apps to Enable SSO for Embedded Content](https://help.tableau.com/current/online/en-us/connected_apps.htm).

Reply:
yikes, Jason!  => JSON